frontend/src/modules/activity/components/activity-timeline.vue

```diff
@@ -12,14 +12,32 @@
           <el-select
             v-model="platform"
             placeholder="All platforms"
+            class="w-40"
           >
+            <template
+              v-if="
+                platform && getPlatformDetails(platform)
+              "
+              #prefix
+            >
+              <img
+                :src="getPlatformDetails(platform).image"
+                class="w-4 h-4"
+              />
+            </template>
```

Comment:
IS this needed? I think you already added it below?

Reply:
Its for selected platform to also show logo